libcrux-ml-dsa/tests/wycheproof_sign.rs

```diff
@@ -50,7 +50,10 @@ macro_rules! wycheproof_sign_test {
                             signature.0.as_slice(),
                             hex::decode(test.sig).unwrap().as_slice()
                         );
-                    } // TODO: else, how should invalid signatures be handled?
+                    }
+                    // TODO: else, the generated signature is invalid; we can
```

Comment:
Can you file an issue to track this?

Reply:
Opened https://github.com/cryspen/libcrux/issues/340 and made a reference to it in the comments.